css/index.css

```
@@ -818,3 +818,605 @@ body {
     gap:48px;
   }
 }
+
+
+/* Mobile-first left*/
+.most-popular {
+  width: 100%;
+  margin: auto;
+  padding: 20px; 
+}
+
+.most-popular__title {
+  font-size: 20px;
+  font-weight: 700;
+  line-height: 24px;
+  letter-spacing: -1%;
+  color: #363535;
+  margin-top: 20px;
+  margin-bottom: 10px;
+}
+
+.most-popular__products {
+  display: flex;
+  flex-direction: column; 
+  gap: 31px;
+}
+
+.most-popular__left {
+  display: flex;
+  flex-direction: column; 
+  gap: 2px;
+  border-radius: 12px;
+}
+
+.most-popular__card,
+.most-popular__cardd,
+.most-popular__carddd,
+.most-popular__cardddd,
+.most-popular__carddddd{
```

Wtf?! 😆 
Si necesitas un estilo diferente para alguna de las card, crea mejor un modifier. Si es para la card 4 haces algo como `most-popular__card--card-4`
```suggestion
.most-popular__card {
```